kotest-property/src/jvmTest/kotlin/com/sksamuel/kotest/property/arbitrary/CollectionsTest.kt

```diff
@@ -91,6 +92,11 @@ class CollectionsTest : DescribeSpec({
             it.shouldHaveAtMostSize(500)
          }
       }
+
+      it("not fail on edge cases for non-empty lists with null values") {
```

Comment:
as the success here is "it doesn't throw an exception", should we say so: `shouldNotThrowAny ...`
also do we need to add any other checks for the output?